doc/developer-notes.md

```diff
@@ -960,6 +960,7 @@ To create a scripted-diff:
 - in the commit message include the bash script between lines containing just the following text:
     - `-BEGIN VERIFY SCRIPT-`
     - `-END VERIFY SCRIPT-`
+- note: you need an extra newline between the above two steps for the scripted diff to work properly
```

Comment:
The newline is generally needed for git to work properly. This is already explained in CONTRIBUTING, see this section:

```
Commit messages should be verbose by default consisting of a short subject line
(50 chars max), a blank line and detailed explanatory text as separate
paragraph(s), unless the title alone is self-explanatory (like "Corrected typo
in init.cpp") in which case a single title line is sufficient. Commit messages should be
helpful to people reading your code in the future, so explain the reasoning for
your decisions. Further explanation [here](https://chris.beams.io/posts/git-commit/).